composer.json

```diff
@@ -53,15 +57,22 @@
         "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
         "package": "Acquia\\Lightning\\Composer\\Package::execute",
         "release-version": "Acquia\\Lightning\\Composer\\ReleaseVersion::execute",
-        "verify-patched-constraints": "Acquia\\Lightning\\Composer\\PatchedConstraint::execute"
+        "verify-patched-constraints": "Acquia\\Lightning\\Composer\\PatchedConstraint::execute",
+        "enable-asset-packagist": "Acquia\\Lightning\\Composer\\AssetPackagist::execute"
     },
     "extra": {
+        "installer-types": [
+            "bower-asset",
+            "npm-asset"
+        ],
         "installer-paths": {
             "docroot/core": [
                 "type:drupal-core"
             ],
             "docroot/libraries/{$name}": [
-                "type:drupal-library"
+                "type:drupal-library",
+                "type:bower-asset",
+                "type:npm-asset"
```

Comment:
> every Lightning user will need to copy and paste those lines

Or run the included [Composer script](https://github.com/acquia/lightning/blob/8.x-2.x/src/Composer/AssetPackagist.php):

```
composer run enable-asset-packagist
```